modules/localgov_media/config/optional/filter.format.wysiwyg.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
   module:
     - editor
     - linkit
+    - localgov_publications
```

Comment:
Not sure it needs, or wants this dependency.

Reply:
Removed